File: mifosng-android/src/main/java/com/mifos/objects/accounts/savings/FieldOfficerOptions.java
package com.mifos.objects.accounts.savings;


/**
 * Created by nkiboi on 12/15/2015.
 */
public class FieldOfficerOptions {
    private Integer id;
    private String firstname;
    private String lastname;
    private String displayName;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    @Override
    public String toString() {
        return "interestCompoundingPeriodType{" +
                "id=" + id +
                ", lastname='" + lastname + '\'' +
                ", firstname='" + firstname + '\'' +
                ", displayName='" + displayName + '\'' +
                '}';
    }
}

File: core/database/src/main/java/com/mifos/core/objects/accounts/savings/FieldOfficerOptions.kt
package com.mifos.core.objects.accounts.savings

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * Created by nkiboi on 12/15/2015.
 */
@Parcelize
class FieldOfficerOptions(
    var id: Int? = null,

    var firstname: String? = null,

    var lastname: String? = null,

    var displayName: String? = null
) : Parcelable